public/i18n/en-us.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
The files in *public/i18n/* are auto-generated using the Gulp task, correct? If that's the case, we should add this folder to our *.gitignore*.